### PR TITLE
Add some options we use elsewhere to these tests

### PR DESCRIPTION
**Pull request by aaomidi**

### CODE REVIEW

File: scripts/test-extensions-unit.bat [hunk 1]
```diff
@@ -21,42 +21,42 @@ if "%ADS_TEST_GREP%" == "" (
 echo ***************************************************
 echo *** starting admin tool extension windows tests ***
 echo ***************************************************
-call .\scripts\code.bat --nogpu --extensionDevelopmentPath=%~dp0\..\extensions\admin-tool-ext-win --extensionTestsPath=%~dp0\..\extensions\admin-tool-ext-win\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --disableExtensions --remote-debugging-port=9222
+call .\scripts\code.bat --nogpu --extensionDevelopmentPath=%~dp0\..\extensions\admin-tool-ext-win --extensionTestsPath=%~dp0\..\extensions\admin-tool-ext-win\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --disable-telemetry --disable-crash-reporter --disable-updates --skip-getting-started --disable-inspect --disable-extensions --remote-debugging-port=9222
```
Comment (Charles-Gagnon) [inline scripts/test-extensions-unit.bat]: So --disable-extensions is fine to use even when testing the extensions themselves? I'm assuming that in this case it still loads the extension passed in on the extensionDevelopmentPath and extensionTestsPath?

Comment (aaomidi) [reply scripts/test-extensions-unit.bat]: Yep, it stops any other extension from loading

**Comment by kburtram** [comment]
@aaomidi what's the thinking with this stale PR?  Do we still want to make this change or should we close this PR?

**Comment by aaomidi** [comment]
I think we can close this out.

**Pull request closed without merging**